pkg/core/region.go

```diff
@@ -855,6 +882,29 @@ func (r *RegionsInfo) PreCheckPutRegion(region *RegionInfo) (*RegionInfo, []*reg
 	return origin, overlaps, err
 }
 
+// GetStaleRegionCnt returns the stale region count.
+func (r *RegionsInfo) GetStaleRegionCnt() int64 {
+	return atomic.LoadInt64(&r.staleRegionCnt)
+}
+
+// AtomicAddStaleRegionCnt atomically adds the stale region count.
+func (r *RegionsInfo) AtomicAddStaleRegionCnt() {
```

Comment:
maybe we can maintain it in the `tree`?

Reply:
I'm not sure what the difference is between putting in `RegionsInfo` and `tree`.
Otherwise, if we put `RegionsInfo` into `tree`, we would need to add a layer to get the `tree`'s `StaleRegion`.

Reply:
changed